compiler/lsp-tests/BUILD.bazel

```diff
@@ -34,6 +35,7 @@ da_haskell_test(
     visibility = ["//visibility:public"],
     deps = [
         "//compiler/damlc/daml-ide-core",
+        "//compiler/damlc/daml-ide-core:hlint-yaml",
```

Comment:
You shouldn’t have to add it to the `deps` of `lsp-tests`. It should come from the runfiles of `damlc`.

Reply:
That's detritus from me trying in desperation debugging. Will remove.

Reply:
Removing.